Octokit/Models/Request/NotificationsRequest.cs

```diff
@@ -21,19 +21,20 @@ public class NotificationsRequest : RequestParameters
         public bool Participating { get; set; }
 
         /// <summary>
-        /// Filters out any notifications updated before the given time. Default: DateTimeOffset.MinValue
+        /// Only show notifications updated after the given time. Defaults to the everything if unspecified.
```

Comment:
Hey @pengwynn, I mentioned this in the chat room, but figured a GitHub comment message was less transient.

According [to the docs](https://developer.github.com/v3/activity/notifications/#list-your-notifications) if you don't specify a `since` it defaults to `Time.now` which doesn't make sense to me as there can't be any notifications since now at the time you make this request.

It makes more sense that `Time.now` is the default for `before`, not `since`. Are the docs wrong?